onlyOffice/src/main/java/org/exoplatform/platform/qa/ui/onlyOffice/pageobject/OnlyOfficeActivityStream.java

```diff
@@ -36,7 +43,21 @@ public void editOnlineFromAS() {
     /**
      * Edit Online button from Preview Document
      */
-    public void editOnlineFromPreviw() {
-        ELEMENT_EDIT_ONlINE_BUTTON.waitUntil(visible,openBrowserTimeoutMs).click();
+    public void editOnlineFromPreview() {
+        ELEMENT_EDIT_ONLINE_BUTTON_FROM_PREVIEW.waitUntil(visible, openBrowserTimeoutMs).click();
     }
+
+    /**
+     * Check alignment of Element
+     * @param element1
+     * @param element2
+     * @param element3
+     */
+    public void checkAlignmentElement(SelenideElement element1, SelenideElement element2, SelenideElement element3) {
+        int locElement1Button = element1.getLocation().y;
+        int locElement2Button = element2.getLocation().y;
+        int locElement3Button = element3.getLocation().y;
+        assertEquals(locElement1Button,locElement2Button,locElement3Button);
+    }
```

Comment:
I don't think it is the role of the functional test to check the pixel position of elements (this must be done by BackstopJS tests). We should only check that the buttons are displayed IMO.

Reply:
but I used the pixel to check just horizontal position (alignment),  also when I did the research I found that it is often used

Reply:
you mean it is often used in our selenium tests ?
That's not the job of these tests IMO.

Reply:
no no, it never used in our selenium